release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -176,6 +176,11 @@ Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plug-in, where
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
 
+[id="ocp-4-12-nw-ipv6-spoke-cluster-support"]
+==== Support for IPv6 spoke clusters from dual-stack hub clusters
+With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the http server on the hub cluster that hosts the boot ISO now listens on both IPv4 address and IPv6 address networks. The provisioning service also checks the BMC address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster.  
```

Comment:
```suggestion
With this update, you can provision IPv6 spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the <what BMC stands for> (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack IPv6 spoke clusters from a dual-stack hub cluster.  
```
A couple of suggestions - I don't think the word "address" is necessary after "IPv4" or "IPv6". What do you think?
I also capitalized HTTP and removed an unnecessary comma.
I'd recommend giving the full name of BMC before using the acronym.

Reply:
Agree with your suggestions. One thing I wasn't sure about was using "address" after the IP formats. I looked at the SSG and it did say use IP address as oppose to IP... which I took it to mean I should say IPv4 address where necessary. Although the SSG entry for IPv4 doesn't say anything about this. 

![image](https://user-images.githubusercontent.com/104497497/200621697-598da735-6496-49ff-b225-eaa47bd9d574.png)

Reply:
Sounds good to me. I think "address" doesn't apply in this case since you're talking about IPv4/6 networks, not individual addresses.

Reply:
Actually I think you are right. I updated it accordingly. Thanks again.